test/RunTests

```diff
@@ -1436,7 +1436,7 @@ printf '3 |f a b c |e x y z\n2 |f a y c |e x\n' | \
     pred-sets/ref/cb_eval1.preds
 
 # Test 124: cb_eval save/load #2
-{VW} -d train-sets/cb_eval2 -i mwt.model -p cb_eval2.preds
+{VW} -d train-sets/cb_eval2 -i mwt.model -p cb_eval2.preds --multiworld_test f
```

Comment:
why is this necessary?

Reply:
not a fan of changing the test, since the thing being tested is changed a bit. Would it be reasonable to infer that info from the test it depends on? see: https://github.com/VowpalWabbit/vowpal_wabbit/blob/07961e3674648a66dd048191be5e8dbec0f6c31d/test/runtests.AUTOGEN.json#L1524

Reply:
Yeah good point, trying it out

Reply:
Restored tests. To get around it I am stashing the original vw command in each test object, and I am having tests use the `depends_on` arguments for file conversions